### PR TITLE
Update bigquery.md with output schemas of V2 destination

### PR DESCRIPTION
**Pull request by ignaciovi**
## What
Due to changes in the BigQuery destination (version 2.0.0+) some parts of the BigQuery documentation are outdated

## How
I've changed the `bigquery.md` file to reflect new airbyte fields populated on the table's output schema

### CODE REVIEW

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
@evantahler can you assignee someone from destination team to validate this doc change?Not sure if we want to reference raw tables in docs instead of the final ones. Or maybe remove how data looks like and only create a link to the destination v2 doc.

File: docs/integrations/destinations/bigquery.md [hunk 1]
```diff
@@ -72,11 +72,11 @@ The BigQuery destination connector supports the following [sync modes](https://d
 
 Airbyte outputs each stream into its own table in BigQuery. Each table contains three columns:
 
-- `_airbyte_ab_id`: A UUID assigned by Airbyte to each event that is processed. The column type in BigQuery is `String`.
-- `_airbyte_emitted_at`: A timestamp representing when the event was pulled from the data source. The column type in BigQuery is `Timestamp`.
-- `_airbyte_data`: A JSON blob representing the event data. The column type in BigQuery is `String`.
+- `airbyte_raw_id`: A UUID assigned by Airbyte to each event that is processed. The column type in BigQuery is `String`.
+- `airbyte_extracted_at`: A timestamp representing when the event was pulled from the data source. The column type in BigQuery is `Timestamp`.
+- `_airbyte_meta`: A JSON blob representing typing errors. You can query these results to audit misformatted or unexpected data. The column type in BigQuery is `JSON`.
```
Comment (evantahler) [inline docs/integrations/destinations/bigquery.md:78]:
```suggestion
... and a column of the proper data type for each of the top-level properties from your source's schema.  Arrays and Objects will remain as JSON columns in BigQuery. 

```

**Review by evantahler — CHANGES_REQUESTED** [review]

File: docs/integrations/destinations/bigquery.md [hunk 2]
```diff
@@ -72,11 +72,12 @@ The BigQuery destination connector supports the following [sync modes](https://d
...
+... and a column of the proper data type for each of the top-level properties from your source's schema.  Arrays and Objects will remain as JSON columns in BigQuery.
```
Comment (evantahler) [inline docs/integrations/destinations/bigquery.md:78]:
```suggestion
... and a column of the proper data type for each of the top-level properties from your source's schema.  Arrays and Objects will remain as JSON columns in BigQuery. Learn more about Typing and Deduping [here](/understanding-airbyte/typing-deduping)
```

**Review by evantahler — APPROVED** [review]
Added some changes, and 👍

**Comment by octavia-approvington** [comment]
Jerry would be proud <br/> Spongebob

**Review by octavia-approvington — APPROVED** [review]

**Pull request merged**